From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Pekka Vuorela <pekka.vuorela@jolla.com>
Date: Thu, 7 Apr 2022 17:26:28 +0300
Subject: [PATCH] Install .prf files as per sailfish setup

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 0120de848ffc9363f6d5ad2b8a3347928556a81c..d94462222c693967b59e9f56a5c485dc334ec325 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -29,7 +29,7 @@ if(NOT DEFINED QT5_PLUGINS_INSTALL_DIR)
 endif()
 
 if(NOT DEFINED QT5_MKSPECS_INSTALL_DIR)
-    set(QT5_MKSPECS_INSTALL_DIR "${CMAKE_INSTALL_LIBDIR}/qt5/mkspecs" CACHE PATH
+    set(QT5_MKSPECS_INSTALL_DIR "${CMAKE_INSTALL_DATAROOTDIR}/qt5/mkspecs" CACHE PATH
             "Installation directory for Qt 5 mkspecs files [LIB_INSTALL_DIR/qt5/mkspecs]")
 endif()
 
